src/app/modules/catalog/containers/catalog-details/catalog-details.component.ts

```diff
@@ -113,8 +113,8 @@ export class CatalogDetailsComponent implements OnInit, OnDestroy {
   buildBooking(): Booking {
     return {
       carId: this.getCarId(),
-      bookedFrom: this.convertNgbDateToDateString(this.startDate!),
-      bookedUntil: this.convertNgbDateToDateString(this.endDate!),
+      bookedFrom: this.createDateFromNgbDate(this.startDate!).toISOString(),
```

Comment:
do you need type assertion?

Reply:
because the type can also be null, but in this case the dates are checked in a method above so they should never be null at this point. If I do not add it it wont compile. I guess I could change the code but honestly I think its too much work for this project